sell_only_by_packaging/models/product_product.py

```diff
@@ -56,5 +56,16 @@ def _convert_packaging_qty(self, qty, uom, packaging):
                 )
                 != 0
             ):
-                qty = qty - (qty % q) + q
+                forced_qty = qty - (qty % q) + q
+                # Support negative rounding
+                if (
+                    float_compare(
+                        qty,
+                        0.0,
+                        precision_rounding=0.001,
```

Comment:
Why this value and not another ?

Reply:
@QuocDuong1306

Reply:
Hi @rousseldenis , I just based on the original one (positive rounding, [here](https://github.com/OCA/sale-workflow/blob/76d07de631e90dd78dbeb3a420bafb18be377af1/sell_only_by_packaging/models/product_product.py#L55)). Do you have any suggestion for this rounding?

Reply:
Yes, of course, using the product rounding.

Reply:
Thank @rousseldenis for your suggestions, a new commit has been added